src/idv_service/idv.service.js

```diff
@@ -229,6 +233,66 @@ class IDVService {
     });
   }
 
+  /**
+   * @param {string} sessionId
+   * @param {CreateFaceCaptureResourcePayload} createFaceCaptureResourcePayload
+   *
+   * @returns {Promise} Resolves CreateFaceCaptureResourceResponse
```

Comment:
```suggestion
   * @returns {Promise<CreateFaceCaptureResourceResponse>}
```

Reply:
Matter of style, elsewhere we had been using the angled brackets syntax to show the resolved type